react-sdk/src/providers/tambo-interactable-provider.tsx

```diff
@@ -297,6 +300,42 @@ export const TamboInteractableProvider: React.FC<PropsWithChildren> = ({
     setInteractableComponents([]);
   }, []);
 
+  const setInteractableStateValue = useCallback(
+    (componentId: string, key: string, value: unknown) => {
+      setInteractableComponents((prev) => {
+        const component = prev.find((c) => c.id === componentId);
+        if (!component) {
+          return prev;
```

Comment:
worth a console.warn that we're updating an interactable that is no (longer? yet?) registered?